### PR TITLE
Add basic project structure

### PR DESCRIPTION
**Pull request by ylembachar**
Added front-end part project setup: 
- React Typescript app
- Using MUI components
- First draft based on the design

### CODE REVIEW

File: frontend/src/routes.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const AppRoutes = () => {
+    return (
+        <Router>
+            <Routes>
+                <Route path="/system-overview" element={<System />} />
+                <Route path="/slot-overview" element={<Slot />} />
+                <Route path="/transaction-detail" element={<Transaction />} />
+            </Routes>
+        </Router>
```
[the remark this answers is not in the document]
Comment (ylembachar) [reply frontend/src/routes.tsx]: Added a 404 page.

File: frontend/src/components/AddressList.js
[the remark this answers is not in the document]
Comment (ylembachar) [reply frontend/src/components/AddressList.js:1]: Updated to tsx.

File: frontend/src/components/BasicTable.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow} from "@mui/material";
+
+interface Transaction {
+    transactionHash: string;
+}
+
+export default function BasicTable() {
+    const [rows, setRows] = React.useState<Transaction[]>([]);
+    const [error, setError] = React.useState<Error | null>(null);
```
[the remark this answers is not in the document]
Comment (ylembachar) [reply frontend/src/components/BasicTable.tsx]: Done.

File: frontend/src/modules/Validator.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    const [totalValidators, setTotalValidators] = useState(null);
+
+    useEffect(() => {
+        // Function to fetch data from the API
+        const fetchData = async () => {
+            try {
+                // Example API endpoints - replace with your actual API endpoints
+                const response1 = await axios.get('/api/shutterizedValidators');
+                const response2 = await axios.get('/api/validatorPercentage');
+                const response3 = await axios.get('/api/totalValidators');
```
[the remark this answers is not in the document]
Comment (ylembachar) [reply frontend/src/modules/Validator.tsx]: Added a custom hook useFetch to fetch the data periodically.

File: frontend/src/components/BasicTable.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (ylembachar) [reply frontend/src/components/BasicTable.tsx]: Updated as well.

**Review by voxil1010 — APPROVED** [review]

**Review by faheelsattar — APPROVED** [review]

**Pull request merged**